Simplenote/Classes/SPNoteListViewController.m

```diff
@@ -229,10 +230,32 @@ - (void)refreshStyle {
 
     // Refresh the SearchBar's UI
     [self.searchBar applySimplenoteStyle];
+
+    self.navigationController.toolbar.translucent = false;
+    self.navigationController.toolbar.barTintColor = [UIColor simplenoteSortBarBackgroundColor];
+}
+
+- (void)refershNavigationButtons {
+    [self updateNavigationBar];
+    [self refreshNavigationControllerToolbar];
+}
+
+- (void)refreshNavigationControllerToolbar
+{
+    if (UIDevice.isPad) {
+        [self.navigationController setToolbarHidden:YES animated:YES];
+    } else {
```

Comment:
Suggestion!!

```
  if (UIDevice.isPad) {
         [self.navigationController setToolbarHidden:YES animated:YES];
         return;
     }
```

That way you can remove the `else` (and unindent a few lines!) 😄